main.go

```diff
@@ -120,9 +138,6 @@ func main() {
 	}
 
 	fmt.Println("=========================================")
-	fmt.Println()
```

Comment:
@syndbg I think you should update the previous line with extra new line in the end to support the same output?

```
fmt.Println("=========================================\n")
```